core/src/test/scala/integration/kafka/api/QuotasTest.scala

```diff
@@ -156,7 +171,7 @@ class QuotasTest extends KafkaServerTestHarness {
                                             RequestKeys.nameForKey(RequestKeys.ProduceKey),
                                             "Tracking throttle-time per client",
                                             "client-id", producerId2)
-    Assert.assertEquals("Should not have been throttled", Double.NaN, allMetrics(producerMetricName).value())
+    assertEquals("Should not have been throttled", Double.NaN, allMetrics(producerMetricName).value(), EPS)
```

Comment:
NaNs are equal, so I think the delta is ignored. In any case, this is changing to a default value of 0 (KAFKA-2567) so it will go away after rebase